hstream-store/HStream/Store/Internal/LogDevice/LogConfigTypes.hs

```diff
@@ -614,9 +614,10 @@ logGroupUpdateExtraAttrs client group extraAttrs =
         let size = logsConfigStatusCbDataSize
             peek_data = peekLogsConfigStatusCbData
             cfun = c_ld_loggroup_update_extra_attrs client' group' l ks' vs'
-        LogsConfigStatusCbData errno version _failure_reason <- withAsync size peek_data cfun
+        (LogsConfigStatusCbData errno version _failure_reason, _) <-
+          withAsync' size peek_data (E.throwSubmitIfNotOK . fromIntegral) cfun
         void $ E.throwStreamErrorIfNotOK' errno
-        syncLogsConfigVersion client version
+        return version
```

Comment:
This is a breaking change, which may lead to test failure.

Reply:
I changed it back. I don't understand what's special about updating the attributes though. It seems that this is the only place where the version has to be synced after we change the state of the log device.